<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSNormalLoginForm</class>
 <widget class="QWidget" name="GBSNormalLoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="imgBanner">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>20</y>
     <width>345</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="btnLoginBiz" native="true">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>204</y>
     <width>501</width>
     <height>548</height>
    </rect>
   </property>
   <widget class="QLabel" name="lblWelcome">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>469</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnScanQRodeLogin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>133</y>
      <width>130</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
		QPushButton:pressed {
		  background-color: #9C1F38;
		  padding-left: 1px; 
		  padding-top: 1px; 
		}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAuthorizeCodeLogin">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>133</y>
      <width>130</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblEmail">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>259</y>
      <width>57</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>邮箱账号</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPassword">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>359</y>
      <width>57</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>登陆密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leEmal">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>280</y>
      <width>500</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 20px;
padding: 5px;
background: white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePassword">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>380</y>
      <width>500</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 20px;
padding: 5px;
background: white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnForgetPassword">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>440</y>
      <width>57</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>忘记密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoginGBS">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>500</width>
      <height>58</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 20px;
padding: 5px;
background: #1B2846;
font-size: 32px;
text-align: center;
</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblNoAccount">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>750</y>
     <width>500</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: transparent;
font-size: 14px; 
text-align: center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>840</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblConstraintAndPrivacy">
   <property name="geometry">
    <rect>
     <x>202</x>
     <y>840</y>
     <width>240</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
